src/universal/config/chapter.ts

```diff
@@ -56,7 +56,7 @@ export const ChapterTitles: { [chapter in Chapter]: string } = {
   ZORG: 'Zorg en ondersteuning',
   VERGUNNINGEN: 'Vergunningen',
   KVK: 'Mijn onderneming',
-  TOERISTISCHE_VERHUUR: 'Toeristische verhuur',
```

Comment:
Dit kun je laten staan. En in je `getToeristischeVerhuurTitle(chapterItem)` functie iets doen als:

```
...
switch (true) {
  case hasVergunningenVakantieVerhuur && !hasVergunningBB:
    return 'Vakantieverhuur';
  case hasVergunningBB && !hasVergunningenVakantieVerhuur:
    return 'Bed & Breakfast';
  default:
   return chapterItem.title;
}
```